e2e/stateful-pallet-storage/handleItemized.test.ts

```diff
@@ -104,7 +105,7 @@ describe('📗 Stateful Pallet Storage', function () {
       const add_actions = [add_action];
       const fake_schema_id = new u16(ExtrinsicHelper.api.registry, 65_534);
       const itemized_add_result_1 = ExtrinsicHelper.applyItemActions(
-        providerKeys,
+        delegatorKeys,
```

Comment:
(you might have already thought about and been waiting before doing this bit)

We'll want to have separate tests for the delegation test vs the direct user test.

Or perhaps with the tests in `e2e/stateful-pallet-storage/handleSignatureRequired.test.ts` should these be reverted to just be testing the delegation with `providerKeys`?

Reply:
I guess it doesn't impact with who is sending it, I was worried that there was a logical difference between the paths for provider vs delegate, but I guess we are safe with either one making the call. The error is the same.